examples/nuxt-3/composables/checkout/index.ts

```diff
@@ -0,0 +1,66 @@
+import { get, set, del } from 'idb-keyval';
+import createShopifyCheckoutClient, {
+  CheckoutClient
+} from '@nacelle/shopify-checkout';
+import { ShopifyCheckout } from '@nacelle/shopify-checkout/dist/types/checkout-client.types';
+import createCheckout from '@nacelle/shopify-checkout/dist/types/client/actions/checkoutCreate';
+import { Ref } from 'vue';
+const checkoutClient: Ref<CheckoutClient | null> = ref(null);
+
+const checkoutState: { checkoutId?: string } = reactive({
+  checkoutId: null
+});
+
+export function useShopifyCheckout(): CheckoutClient {
+  if (!checkoutClient.value) {
+    const runTimeConfig = useRuntimeConfig();
+    checkoutClient.value = createShopifyCheckoutClient({
+      storefrontCheckoutToken:
+        runTimeConfig.public.shopify.StorefrontCheckoutToken,
+      myshopifyDomain: runTimeConfig.public.shopify.myShopifyDomain
+    });
+  }
+  return reactive(checkoutClient.value);
+}
+
+export async function initCheckout(): Promise<void> {
+  try {
+    const shopifyCheckoutClient = useShopifyCheckout();
+    checkoutState.checkoutId = await get('checkoutId');
+    if (checkoutState.checkoutId) {
+      const checkout = await shopifyCheckoutClient.get({
+        id: checkoutState.checkoutId
+      });
```

Comment:
<img width="600" alt="Property 'completed' does not exist on type 'void'" src="https://user-images.githubusercontent.com/5732000/185224935-7872d59a-94c8-4e97-b5c9-854d503fe336.png">

```suggestion
      const checkout = (await shopifyCheckoutClient.get({
        id: checkoutState.checkoutId
      })) as ShopifyCheckout;
```